tslint.js

```diff
@@ -1,79 +1,91 @@
 const path = require("path");
 
+// NOTE: Do not add the following rules (see
+// https://github.com/ionic-team/tslint-ionic-rules/commit/065e3e9937104f6ec03e594adf762002616353df):
+// - no-unnecessary-type-assertion
+// - prefer-for-of
+// - no-conditional-assignment
+
 module.exports = {
   "extends": ["tslint-eslint-rules"],
   "rulesDirectory": path.join(path.dirname(require.resolve("tslint-eslint-rules")), "dist/rules"),
   "rules": {
+    "adjacent-overload-signatures": true,
     "align": true,
+    "array-type": [true, "array"],
+    "arrow-return-shorthand": true,
     "await-promise": true,
     "ban-comma-operator": true,
     "class-name": true,
     "comment-format": {"options": ["check-space"]},
+    "curly": [true, "ignore-same-line"],
+    "deprecation": { "options": true, "severity": "warning" },
     "encoding": true,
     "eofline": true,
-    "indent": {"options": ["spaces"]},
+    "import-spacing": true,
+    "indent": {"options": ["spaces", 2]},
+    "interface-over-type-literal": true,
     "jsdoc-format": [true, "check-multiline-start"],
     "linebreak-style": [true, "LF"],
+    "member-access": [true, "no-public"],
     "new-parens": true,
+    "no-arg": true,
+    "no-constant-condition": true,
+    "no-construct": true,
+    "no-control-regex": true,
     "no-debugger": true,
+    "no-duplicate-case": true,
+    "no-duplicate-imports": true,
     "no-duplicate-super": true,
     "no-duplicate-switch-case": true,
     "no-duplicate-variable": true,
     "no-empty": true,
+    "no-empty-character-class": true,
     "no-eval": true,
+    "no-ex-assign": true,
     "no-extra-boolean-cast": true,
+    "no-extra-semi": true,
     "no-import-side-effect": true,
     "no-inferrable-types": true,
+    "no-inner-declarations": true,
     "no-internal-module": true,
+    "no-invalid-regexp": true,
+    "no-invalid-template-strings": true,
+    "no-irregular-whitespace": true,
     "no-misused-new": true,
+    "no-multi-spaces": true,
     "no-non-null-assertion": true,
+    "no-redundant-jsdoc": true,
+    "no-reference": true,
+    "no-regex-spaces": true,
+    "no-return-await": true,
+    "no-shadowed-variable": { "options": true, "severity": "warning" },
```

Comment:
I saw this one a lot in Framework, so I made it a warning. It'd probably be best to turn it into an actual lint error sometime.

https://palantir.github.io/tslint/rules/no-shadowed-variable/